src/components/organisms/AssetActions/Pool/Remove.module.css

```diff
@@ -60,3 +60,28 @@
 .output figure[class*='pool shares'] {
   display: none;
 }
+
+.output p {
+  font-size: var(--font-size-small);
+}
+
+.slippage {
+  composes: slippage from '../Trade/Slippage.module.css';
+}
+
+.slippage select {
```

Comment:
this is redundant, because it will be inherited from the above `compose`. So we can remove all declarations here for everything which is already defined in `../Trade/Slippage.module.css`:
-  `.slippage select`
- `.slippage strong`

And since we do not have any `input` in this component, `.slippage input` can be removed too